crates/matrix-sdk-base/src/client.rs

```diff
@@ -1403,6 +1403,15 @@ impl BaseClient {
         self.store.room(room_id)
     }
 
+    /// Forget the room with the given room ID.
```

Comment:
Might be useful to explain how it differs from leaving a room?

Reply:
Is what I added okay?

Reply:
But it's sending particular request. I think a user may not want to read the spec to understand the difference between `room.leave()` and `room.forget()`. Can we clarify that please?

Reply:
This is the base SDK, there is not request.